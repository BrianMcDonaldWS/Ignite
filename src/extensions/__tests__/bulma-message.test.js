import MarkdownIt from 'markdown-it';
import message from '../bulma-message';

const md = new MarkdownIt().use(message);

test('bulma - message', () => {
  expect(
    md.render(`
::: message is-danger Message with a Title
Body of the message
:::
  `)
  ).toMatchSnapshot();
});
