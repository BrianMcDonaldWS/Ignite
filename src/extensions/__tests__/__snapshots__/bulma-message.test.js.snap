// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`bulma - message 1`] = `
"
        <article class=\\"message column is-danger\\">
          
            <div class=\\"message-header\\">
              <p>Message with a Title</p>
            </div>
          
          <div class=\\"message-body\\">
      <p>Body of the message</p>

        </div>
      </article>
    "
`;
